### PR TITLE
Rdm 9478 fix

### PR DESCRIPTION
**Pull request by henrykcgi**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-RDM-9052.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<!-- from version 3.7.0 onwards liquibase doesn't create the sequence unless the autoincrement is defined-->    
+    <changeSet id="RDM-9052-1" author="henry">
+	    <preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">select count (a.relname) from pg_class a where a.relname ='user_profile_audit' and Not Exists (SELECT c.relname FROM pg_class c WHERE c.relkind = 'S' and c.relname = 'user_profile_audit_id_seq')</sqlCheck>
+	    </preConditions>
+		<addAutoIncrement columnDataType="int" columnName="id" tableName="user_profile_audit"/>  
+	</changeSet>
+</databaseChangeLog>
+
+
```
Comment (hemantt) [inline src/main/resources/db/changelog/db.changelog-RDM-9052.xml:14]: We might need to remove this RDM-9052 change? @henrykcgi

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -8,4 +8,5 @@
     <include file="db/changelog/db.changelog-RDM-2761-UserProfileAuditTable.xml"/>
     <include file="db/changelog/db.changelog-RDM-3853.xml"/>
     <include file="db/changelog/db.changelog-RDM-5499.xml"/>
+    <include file="db/changelog/db.changelog-RDM-9052.xml"/>
```
Comment (hemantt) [inline src/main/resources/db/changelog/db.changelog-master.xml:11]: We might need to remove this RDM-9052 change? @henrykcgi

**Review by hemantt — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/db.changelog-master.xml (hunk 2 above)
Comment (henrykcgi) [reply src/main/resources/db/changelog/db.changelog-master.xml:11]: No Not really.. it is the fix and required.

File: src/main/resources/db/changelog/db.changelog-RDM-9052.xml (hunk 1 above)
Comment (henrykcgi) [reply src/main/resources/db/changelog/db.changelog-RDM-9052.xml:14]: not really it is the fix.

**Pull request merged**